templates/customize-adventure.html

```diff
@@ -209,4 +210,13 @@ <h3>{{_('adventure_exp_classes')}}</h3>
         </div>
     </div>
 </div>
+<script>
+    document.addEventListener("DOMContentLoaded", () => {
```

Comment:
A while ago we were trying to get away from adding scripts to the templates, because we wanted all javascript code to be in a single place. I think you could use `initiliaze.ts` to initialize these functions in JavaScript.

Reply:
Why though? I mean we don't really need to initialize anything if the route is not in the wanted page! What I do know is that the main issues with having different scripts is that it could cause a sluggish experience. Was there another (specific) problem we faced? I see that we still use it in some places!! 

We could put it in initialize.ts, but it's not as incapsulated as this is! Perhaps it's a matter of docummentation?

Reply:
I think the reason is that if all code is in one place, it is easy to see what is there. In the past we ran into the issue that similar code was in multiple places and when something was changed, some locations were overlooked. People might not look for js inside an HTML file (it is a sort of separation of concerns)

So I support the idea of having as much in one place as possible!

Reply:
I agree about the separation of cencerns concern. Will change it. 

Two points. First, so the current scripts that are in html files we still have to clean up? And second there's several downsides of our current approach. The main and crucial one is that gradually more and more TS must be loaded even though it's not in use yet. This affects the overall performance of our app. If you're satisfied with the overall performance, we tackle it when needed. Just good to keep it in mind i think!

Reply:
> Two points. First, so the current scripts that are in html files we still have to clean up? 

Yes, I would assume so! There is a lot of places where we still have js/ts in the HTML files, and we like to move that to HTMX or to a typescript file. I don't necessarily think we should do all of that right away, but maybe if you encounter it while working on other things, you can try to clean it up piece by piece. You can also do a bit of cleanup in between other tasks if you think it is a fun palet cleanser!

> The main and crucial one is that gradually more and more TS must be loaded even though it's not in use yet.

I don't think that is where the bottle neck of performance is, but yes let's keep it in mind!